fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Avatar.kt

```diff
@@ -215,6 +208,83 @@ fun Avatar(
     }
 }
 
+@Composable
```

Comment:
is this supposed to be exposed to public, if not can we make it internal or private?

Reply:
done